### ISSUE TITLE
Training loss is negative

### ISSUE THREAD

**Issue by JH-ninjatech** [sihyun-yu/REPA#13]
Hi @sihyun-yu,

I have applied the idea of your work on my DiT model and found that my training loss is becoming negative after several training steps. Did you see similar situation in your model training and is this normal? Do I need to scale down the value of `proj-coeff`?

**Comment by sihyun-yu** [comment]
Hi, it is natural because they are defined as a negative cosine similarity.
